munit/shared/src/main/scala/munit/ScalaCheckEffectSuite.scala

```diff
@@ -85,23 +81,14 @@ trait ScalaCheckEffectSuite extends ScalaCheckSuite {
     }
   }
 
-  private def fixResultException(result: Test.Result): Test.Result =
+  private def fixResultException(result: ScalaCheckTest.Result): ScalaCheckTest.Result =
     result.copy(
       status = result.status match {
-        case p @ PropException(_, e, _) => p.copy(e = rootCause(e))
+        case p @ PropException(_, e, _) => p.copy(e = Exceptions.rootCause(e))
         case default                    => default
       }
     )
 
-  // https://github.com/scalameta/munit/blob/68c2d13868baec9a77384f11f97505ecc0ce3eba/munit/shared/src/main/scala/munit/MUnitRunner.scala#L318-L326
-  @tailrec
-  private def rootCause(x: Throwable): Throwable = x match {
-    case _: InvocationTargetException | _: ExceptionInInitializerError |
-        _: UndeclaredThrowableException | _: ExecutionException if x.getCause != null =>
-      rootCause(x.getCause)
-    case _ => x
-  }
```

Comment:
This is now a public method in MUnit.
- https://github.com/scalameta/munit/pull/542